src/services/navigation.ts

```diff
@@ -4,5 +4,6 @@ export type {
   Waypoint,
   NavigationPosition,
   Path,
+  NavigationProperties,
```

Comment:
If we're exporting NavigationProperties, we should also export this in `src/main.ts`.